simple_geo3x3.el

```diff
@@ -1,16 +1,15 @@
-(setq load-path nil)
-(load "geo3x3")
+;;; Code:
 
-(message (geo3x3_encode 35.65858 139.745433 14))
+(require 'geo3x3 (expand-file-name "geo3x3" default-directory))
```

Comment:
By using `require` instead of `load`, functions can be loaded using [named features](https://www.gnu.org/software/emacs/manual/html_node/elisp/Named-Features.html) ([translated in Japanese](https://ayatakesi.github.io/emacs/24.5/elisp_html/Named-Features.html)).

Reply:
Thank you! LGTM!

<img width="232" alt="image" src="https://github.com/taisukef/Geo3x3/assets/1715217/8cb81eeb-6214-4b63-869c-2f7864288a3e">

Do this character [ff] need for Emacs Lisp code?

Reply:
Although this character is not required, Emacs recognizes `^L` (ASCII 12, [Form Feed](https://en.wikipedia.org/wiki/Page_break#Form_feed)) as a [page break character](https://www.gnu.org/software/emacs/manual/html_node/emacs/Pages.html), so it is conventionally used as a section separator.